### DISCUSSION TITLE
Payment methods don't allow more than 5 options

### DISCUSSION THREAD

**Discussion by sahilc0** [RoboSats/robosats#1763] [Ideas]
**Describe the bug**
Payment methods throws an error if selecting more than 5 options. Is this expected behavior?
<img width="356" alt="CleanShot 2023-01-01 at 15 38 11@2x" src="https://user-images.githubusercontent.com/8256062/210185310-3cf8a56e-0db7-4159-ad15-aa03582c4091.png">

**Expected behavior**
Users should be able to select as many options as possible, to increase chances that other peers can trade with them

**Comment by Reckless-Satoshi** [comment]
The payment methods field is restricted by length of the concatenated string (sum of characters of every payment method). I believe it is currently limited to 50 characters, slightly less than the max length accepted by the coordinator API. That means, it is possible to add more payment methods, but only if they name of the methods is short.

In any case, this opens up an interesting discussion. How many payment methods are considered sufficient? I envisioned the current set up to be enough, but maybe I felt short :)
